src/components/HomeSocialPresence.astro

```diff
@@ -38,14 +38,13 @@ import IconLinkedin from "./icons/icon-linkedin.astro"
     </SocialCard>
 
     <SocialCard
-        href="https://www.facebook.com/groups/frontendcodersmu"
-        class="bg-[#4267B2] text-sm md:text-normal"
+        href="https://chat.whatsapp.com/CqXAj6m8NtQB6NN8B9aoEJ"
+        class="bg-[#25D366] text-sm md:text-normal"
     >
-        <p slot="tagline">Facebook Group</p>
-        <p slot="description">Our legacy presence on Facebook.</p>
-        
-        <IconFacebookGroup class="w-14 md:w-16 p-2" slot="icon"/>
+        <p slot="tagline">Whatsapp Group</p>
+        <p slot="description">Our Community on Whatsapp</p>
```

Comment:
Change the text to : 

`Join this community to stay informed about our events`